src/lib/storage/chunk.cpp

```diff
@@ -26,9 +24,9 @@
 
 namespace hyrise {
 
-Chunk::Chunk(Segments segments, const std::shared_ptr<MvccData>& mvcc_data,
-             const std::optional<PolymorphicAllocator<Chunk>>& alloc, Indexes indexes)
-    : _segments(std::move(segments)), _mvcc_data(mvcc_data), _indexes(std::move(indexes)) {
+Chunk::Chunk(Segments segments, const std::shared_ptr<MvccData>& mvcc_data, PolymorphicAllocator<Chunk> alloc,
```

Comment:
Why no longer an optional?

Reply:
Because `std::pmr::polymorphic_allocator`'s assignment operator (`operator=`) is deleted. Therefore, `_alloc = *alloc;` as done before will not work. Before, when no allocator is passed, `_alloc` is default initialized as well.